gr4vy/gr4vy_client.py

```diff
@@ -29,9 +30,10 @@
 
 
 class Gr4vyClient:
-    def __init__(self, gr4vyId, private_key_file):
+    def __init__(self, gr4vyId, private_key_file, environment):
```

Comment:
Should this default to `production`?

Reply:
I think it effectively does?

Reply:
in configuration if anything but production is passed as environment it will do *environment*.*gr4vyid* for base url. Thing this is safe as prevents only implied production calls